--- src/prop_ext.rs	2006-07-24 04:21:28
+++ src/prop_ext.rs	2026-07-06 09:53:58
@@ -219,12 +219,10 @@
     /// Get the patched highlight style with focused or unfocused style.
     #[inline]
     pub fn get_style_focus(&self, normal_style: Style, focus: bool) -> Style {
-        let style = normal_style.patch(self.style);
-
         if focus {
-            style
+            normal_style.patch(self.style)
         } else {
-            style.patch(self.style_unfocused)
+            self.style_unfocused
         }
     }
 }
